fixtures/fixture.py

```diff
@@ -91,9 +91,9 @@ def setup_test_repo():
         #     ('python3 ../pantheon.py --user={} --password={} --server={} push'.format(uploader_username,
         #                                                                               uploader_password,
         #                                                                               url)), shell=True)
-        subprocess.check_call(
-            ('pantheon --user={} --password={} --server={} push'.format(uploader_username, uploader_password,
-                                                                        url)), shell=True)
+        #subprocess.check_call(
```

Comment:
Please uncomment this
We had commented this out only while trying out the tests